UndertaleModLib/Models/UndertaleEmbeddedTexture.cs

```diff
@@ -35,10 +35,16 @@ public class UndertaleEmbeddedTexture : UndertaleNamedResource, IDisposable
     public uint GeneratedMips { get; set; }
 
     /// <summary>
-    /// TODO: something. <br/>
+    /// The size of the texture data in the embedded image. <br/>
     /// GameMaker: Studio 2 only.
     /// </summary>
-    public uint TextureBlockSize { get; set; }
+    private uint _TextureBlockSize { get; set; }
+
+    /// <summary>
+    /// The position of the TextureBlockSize value to be overwritten in Serialize. <br/>
```

Comment:
Adress TextureBlockSize via cref

Reply:
```suggestion
    /// The position of the placeholder <see cref="_TextureBlockSize">TextureBlockSize</see> value
    /// to be overwritten in Serialize. <br/>
```
istg if this sent earlier